.svelte-kit/tsconfig.json

```diff
@@ -27,12 +27,16 @@
 	},
 	"include": [
 		"ambient.d.ts",
+		"../vite.config.ts",
```

Comment:
I wonder if we can just remove the `includes` property as includes should include everything not in `exclude`. maybe theres more of a reason for include than I realize though.

Reply:
This is a file controlled by SvelteKit. If we made any changes here, I'm sure they'd get overwritten.